### ISSUE TITLE
Add various layer types

### ISSUE THREAD

**Issue by dhruvrajan** [dhruvrajan/tensorflow-keras-java#6]
- [ ] Conv2D
- [ ] LSTM
- [ ] Dropout
- [ ] Flatten
- [ ] MaxPool2D

**Comment by zaleslaw** [comment]
I could help with Conv2D or different pooling for example (implemented them in private project)

**Comment by Sciss** [comment]
How difficult is this to implement? I'm looking for

- [ ] BatchNormalization
- [ ] LeakyReLU
- [ ] Reshape
- [ ] Conv2DTranspose
- [ ] Dropout

**Comment by zaleslaw** [comment]
@Sciss please have a look at the [KotlinDL](https://github.com/JetBrains/KotlinDL) library
